src/appMain/hmi_capabilities.json

```diff
@@ -1,439 +1,678 @@
+            "upDownAvailable": false
+          },
+          {
+            "name": "FAN_DOWN",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "TEMP_UP",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "TEMP_DOWN",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "DEFROST_MAX",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "DEFROST",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "DEFROST_REAR",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "UPPER_VENT",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "LOWER_VENT",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "VOLUME_UP",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "VOLUME_DOWN",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "EJECT",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "SOURCE",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "SHUFFLE",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          },
+          {
+            "name": "REPEAT",
+            "shortPressAvailable": true,
+            "longPressAvailable": true,
+            "upDownAvailable": false
+          }
+        ]
     },
-    "SyncMessageVersion": {
-        "majorVersion": 3,
-        "minorVersion": 0
+    "videoStreamingCapability": {
+      "preferredResolution": {
+        "resolutionWidth": 800,
+        "resolutionHeight": 350
+      },
+      "maxBitrate": 10000,
+      "supportedFormats": [
+        {
+          "protocol": "RAW",
+          "codec": "H264"
+          }
+        ]
```

Comment:
Be prepared for these formatting changes to cause conflicts once #1712 is merged. Shouldn't be hard to resolve though